draft-ietf-acme-acme.md

```diff
@@ -894,6 +904,7 @@ Content-Type: application/jose+json
     "url": "https://example.com/acme/new-reg"
   })
   "payload": base64url({
+    "external_secret": "XYZ123",
```

Comment:
This was my mistake. I copied the `contact` field description but failed to edit it appropriately. Fixed now.